src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160920_attribute_namespace.xml

```diff
@@ -34,6 +34,16 @@
             create temporary table WORKSPACE_ATTRIBUTE_TEMP (id bigint(20) unsigned NOT NULL AUTO_INCREMENT primary key, namespace text NOT NULL, name text NOT NULL, value_string text, value_number double DEFAULT NULL, value_boolean bit(1) DEFAULT NULL, value_entity_ref bigint(20) unsigned DEFAULT NULL, list_index int(11) DEFAULT NULL, list_length int(11) DEFAULT NULL, owner_id binary(16) NOT NULL, transaction_id CHAR(36) NOT NULL);
             END
         </createProcedure>
+    </changeSet-->
+
+    <changeSet id="ENTITY_ATTRIBUTE_TEMP_real" author="dvoet">
```

Comment:
Since they are no longer temp tables, we should rename them.  working tables?  scratch tables?